nsis/src/nsis/NsisLoader.java

```diff
@@ -47,32 +47,40 @@
 import nsis.format.NsisScriptHeader;
 
 public class NsisLoader extends PeLoader {
-	
-	public final static String NE_NAME = "NSIS Executable (NE)";
+
+	public final static String NE_NAME = "NSIS Executable (PE + NSIS)";
 
 	@Override
 	public String getName() {
 		return NE_NAME;
 	}
 
 	@Override
-	public Collection<LoadSpec> findSupportedLoadSpecs(ByteProvider provider) throws IOException { //TODO call super to handle PE portion of the file
+	public Collection<LoadSpec> findSupportedLoadSpecs(ByteProvider provider)
+			throws IOException { // TODO call super to handle PE portion of the
+									// file
 		List<LoadSpec> loadSpecs = new ArrayList<>();
-		NsisExecutable ne = NsisExecutable.createNsisExecutable(RethrowContinuesFactory.INSTANCE, provider, SectionLayout.FILE);
-		if(ne.getHeaderOffset() != 0) {
-			LoadSpec my_spec = new LoadSpec(this, 0x400000, new LanguageCompilerSpecPair("Nsis:LE:32:default", "default"), true);
+		NsisExecutable ne = NsisExecutable.createNsisExecutable(
+				RethrowContinuesFactory.INSTANCE, provider, SectionLayout.FILE);
+		if (ne.getHeaderOffset() != 0) {
```

Comment:
The default value of  `header_offset` in NsisExecutable is -1, I think this value should be -1 as well.

Reply:
Good catch, I changed it